tests/std/tests/P0896R4_views_single/test.cpp

```diff
@@ -200,7 +200,11 @@ struct ConstSelection {
     int value = 0;
 };
 
+#ifdef __EDG__ // TRANSITION, VSO-1898912
+static_assert(!is_trivially_copy_assignable_v<ranges::single_view<ConstSelection>>);
+#else // ^^^ assert bug so we'll notice when it's fixed / no workaround vvv

```

Comment:
No change requested: I personally dislike this pattern (it interferes with picking up fixes) but I won't object to it because there are pre-existing occurrences.